resources/istio/templates/configmap-control-plane.yaml

```diff
@@ -8,14 +8,14 @@ data:
   config.yaml: |-
 {{ tpl (.Files.Get "files/kyma_minikube_istio_operator.yaml" | printf "%s" | indent 4) . }}
   operator-1-7.yaml: |-
-{{ tpl (.Files.Get "files/istio-operator-1-7-minikube.yaml" | printf "%s" | indent 4) . }}
+{{ tpl (tpl ("files/{{ .Values.istio_operator_minikube_file }}") . | .Files.Get | printf "%s" | indent 4) . }}
```

Comment:
I don't understand, why the double `tpl` ?

Reply:
otherwise, it doesn't work. funny thing istio is installed with default values and patching webhook doesn't work

Reply:
the "internal" `tpl` is required to parse the `{{ .Values.istio_operator_minikube_file }}` value in the file path